Composer/cypress/integration/NotificationPage.spec.ts

```diff
@@ -71,7 +71,7 @@ context('Notification Page', () => {
         .wait(1000);
     });
 
-    cy.get('[data-testid="notifications-info-button"]').click();
+    cy.get('[data-testid="LeftNav-CommandBarButtonNotifications"]').click();
```

Comment:
Can I get you to update these to the testing library method?

`cy.findByTestId(id)`

Reply:
Will do. Thanks for reminding me of that.